pkg/security/module/server.go

```diff
@@ -231,17 +231,29 @@ func (a *APIServer) dequeue(now time.Time, cb func(msg *pendingMsg) bool) {
 	})
 }
 
-func (a *APIServer) updateMsgTags(msg *api.SecurityEventMessage, includeGlobalTags bool) {
-	// on fargate, append global tags
-	if includeGlobalTags && fargate.IsFargateInstance() {
-		for _, tag := range a.getGlobalTags() {
-			key, _, _ := strings.Cut(tag, ":")
-			if !slices.ContainsFunc(msg.Tags, func(t string) bool {
-				return strings.HasPrefix(t, key+":")
-			}) {
-				msg.Tags = append(msg.Tags, tag)
+func (a *APIServer) updateMsgTags(msg *api.SecurityEventMessage, isCustomEvent bool) {
+	if isCustomEvent {
```

Comment:
I would suggest to remove the isCustomEvent bool and put the if condition when call this function

Reply:
suggestion applied